src/30_Advanced/Basic_User_Consent_Flow.html

```diff
@@ -0,0 +1,162 @@
+      width: 300px;
+      position: relative;
+    }
+
+    #ui1 {
+      position: fixed;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      background: rgba(0, 0, 0, 0.7);
+    }
+
+    amp-consent button {
+      border: none;
+      border-radius: 2px;
+      color: #fafafa;
+      height: 26px;
+      min-width: 32px;
+      padding: 0 16px;
+      margin: 0 16px;
+      text-transform: uppercase;
+      letter-spacing: 0;
+      cursor: pointer;
+      vertical-align: middle;
+      line-height: 26px;
+      text-align: center;
+      background: #3f51b5;
+    }
+
+    .buttonbar {
+      padding-top: 15px;
+      padding-bottom: 15px;
+    }
+
+  </style>
+</head>
+<body>
+
+  <!---{
+  "experiments": ["amp-consent"]
+  }--->
+
+  <!-- ## Important -->
+  <!--
+  The consent is saved to localstorage - after accepting/rejecting it once you will
+  only get the popup dialog again after deleting the localstorage content for this host.
+
+  -->
+
+  <!-- ## Basic Usage -->
+  <!--
+  The amp-consent component allows you:
+  * To ask a remote endpoint if consent is required
+  * Specify the UI to trigger the consents
+  * Specify the policy to be used for this consent
+  * You can dynamically load in the vendors to get consent for via amp-list
+  -->
+  <amp-consent id='amp-consent' layout='nodisplay'>
+        <script type="application/json">{
+          "consents": {
+            "DEF": {
+              "checkConsentHref": "/samples_templates/consent/getConsent",
+              "promptUI": "ui1"
+            }
+          },
+          "policy": {
+            "Policy_DEF": {
+              "waitFor": {
+                "DEF": []
+              }
+            }
+          }
+        }</script>
+        <div id="ui1">
+          <div id="consentPopup">
+             It is always good to give users the choice on data sharing.
+             Do you want to share your data with these 3rd parties?
+             <center>
+               <amp-list width="auto"
+                height="100"
+                layout="fixed-height"
+                src="/samples_templates/consent/getVendors">
+                <template type="amp-mustache">
+                  <div>
+                    {{name}}
+                  </div>
+                </template>
+              </amp-list>
+            </center>
+             <div class="buttonbar">
+               <button on="tap:ui1.hide,amp-consent.accept" >
+                  Accept
+              </button>
+              <button on="tap:ui1.hide,amp-consent.reject" >
+                  Reject
+              </button>
+          </div>
+          </div>
+        </div>
+      </amp-consent>
+
+    <!--
+    You can block content until consent is given by setting the data-block-on-consent
+    attribute to any AMP component.
+    Here is a blocked image.
+    -->
+    <amp-img data-block-on-consent src="/img/landscape_lake_1280x857.jpg" width="1280" height="857" layout="responsive">
```

Comment:
It's better to give the image a fixes size (the responsive layout results in massive images on large desktops)